csunplugged/topics/views.py

```diff
@@ -198,13 +197,19 @@ def get_context_data(self, **kwargs):
         context["lesson"] = lesson
         context["unit_plan"] = lesson.unit_plan
         context["topic"] = lesson.topic
-        for programming_challenge in context["programming_challenges"]:
+        programming_challenges = context["programming_challenges"]
+        for programming_challenge in programming_challenges:
             challenge_numbers = ProgrammingChallengeNumber.objects.get(
                 lesson=lesson,
                 programming_challenge=programming_challenge
             )
             programming_challenge.challenge_set_number = challenge_numbers.challenge_set_number
             programming_challenge.challenge_number = challenge_numbers.challenge_number
+        context["programming_challenges"] = programming_challenges.order_by(
+            "challenge_set_number",
+            "challenge_number",
+            "name",
```

Comment:
aren't they already sorted by set number and number? not sure that name really adds anything here? (or am I missing something?)